contracts/rewards/RewardsManager.sol

```diff
@@ -28,11 +28,10 @@ import "./IRewardsManager.sol";
  * These functions may overestimate the actual rewards due to changes in the total supply
  * until the actual takeRewards function is called.
  */
-contract RewardsManager is RewardsManagerV3Storage, GraphUpgradeable, IRewardsManager {
+contract RewardsManager is RewardsManagerV4Storage, GraphUpgradeable, IRewardsManager {
     using SafeMath for uint256;
 
-    uint256 private constant TOKEN_DECIMALS = 1e18;
-    uint256 private constant MIN_ISSUANCE_RATE = 1e18;
+    uint256 private constant FIXED_POINT_SCALING_FACTOR = 1e18;
```

Comment:
This name is more precise

Reply:
Yeah, the auditors had commented TOKEN_DECIMALS could be confusing in #571 so I used this name there as well